includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -791,6 +791,29 @@ public function return_handler() {
 		parent::return_handler();
 	}
 
+	/**
+	 * Check is Capture possible
+	 *
+	 * @param WC_Order|int $order
+	 * @param bool $amount
+	 *
+	 * @return bool
+	 */
+	public function can_capture( $order, $amount = false ) {
```

Comment:
I can't see that `can_capture()` is used anywhere? Also, the `href` from the operation should be returned somehow, otherwise the URL will have to be built (which is a big no) or the operation and its `href` will have to be retrieved one more time.

Reply:
@asbjornu ```can_capture``` used in the parent the payments module. We have those feature including such UI over there.

Reply:
That doesn't make any sense, since `create-paymentorder-capture` is specifically for `paymentorder` (and thus Checkout) and not available in any of the payment instruments in Payments. Anyhow, the method still doesn't make much sense since it only returns `true`. What is supposed to happen when  `can_capture` returns `true`? Where will the `href` of the `create-paymentorder-capture` come from? How would anyone invoking `can_capture()` know which URL they should `POST` to in order to perform the capture?